<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="4">
     <widget class="QPushButton" name="zCCWRotationPushButton">
      <property name="text">
       <string>OZ CCW</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLabel" name="rotationLabel">
      <property name="text">
       <string>                 Rotation</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="yCWRotationPushButton">
      <property name="text">
       <string>OY CW</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="xCWRotationPushButton">
      <property name="text">
       <string>OX CW</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="openFilePushButton">
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="3">
     <widget class="QLabel" name="movementLabel">
      <property name="text">
       <string>              Movement</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QPushButton" name="rightMovementPushButton">
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8" colspan="2">
     <widget class="QLabel" name="normalizationLabel">
      <property name="text">
       <string>             Normalization</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="yCCWRotationPushButton">
      <property name="text">
       <string>OY CCW</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLabel" name="scalingLabel">
      <property name="text">
       <string>                  Scaling</string>
      </property>
     </widget>
    </item>
    <item row="8" column="7">
     <widget class="QPushButton" name="forwardMovementPushButton">
      <property name="text">
       <string>Forward</string>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <widget class="QLabel" name="toNormalizationLabel">
      <property name="text">
       <string>         To</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="zCWRotationPushButton">
      <property name="text">
       <string>OZ CW</string>
      </property>
     </widget>
    </item>
    <item row="5" column="9">
     <widget class="QLineEdit" name="toNormalizationLineEdit"/>
    </item>
    <item row="3" column="8">
     <widget class="QLabel" name="fromNormalizationLabel">
      <property name="text">
       <string>      From</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QLineEdit" name="errorLineEdit"/>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="leftMovementPushButton">
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="backMovementPushButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="minusScalingPushButton">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="xCCWRotationPushButton">
      <property name="text">
       <string>OX CCW</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLineEdit" name="fileNameLineEdit"/>
    </item>
    <item row="0" column="0" colspan="10">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="loadDataPushButton">
      <property name="text">
       <string>Load Data</string>
      </property>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QLineEdit" name="fromNormalizationLineEdit"/>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="plusScalingPushButton">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="3">
     <widget class="QPushButton" name="upMovementPushButton">
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5" colspan="3">
     <widget class="QPushButton" name="downMovementPushButton">
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
